package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "next build",
+    "build": "npm run delete:nostr-relaypool-ts && next build",
```

Comment:
This is necessary to fix next build.

There is an issue nostr-relaypool; it publishes all the source ts files that don't play nicely with next.js build step.